examples/functions-with-defaultable-parameters/functions_with_defaultable_parameters.bal

```diff
@@ -1,13 +1,13 @@
 import ballerina/io;
 
-// This function takes salary, annual increment, and bonus rate as input 
+// This function takes base salary, annual increment and bonus rate as input
```

Comment:
```suggestion
// This function takes base salary, annual increment, and bonus rate as input
```

The serial comma usage is correct here actually. You would normally have the comma in front of "and" in a list.